ci/test_unit.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Get latest Transformers4Rec version
+cd /transformers4rec/
+git pull origin main
+
+container=$1
+
+## Tensorflow container
+if [ "$container" == "merlin-tensorflow-training" ]; then
+    pytest tests/tf
+# Pytorch container
+elif [ "$container" == "merlin-pytorch-training" ]; then
+    pytest tests/torch
+# Inference container
+elif [ "$container" == "merlin-inference" ]; then
+    pytest tests
```

Comment:
Do we need to differentiate between the containers on running the unittests? 

The transformers4rec tests here do things like ```torch = pytest.importorskip("torch")``` to skip if we don't have torch or tensorflow installed. (also I don't think the inference container has TF - so we're already relying on this behaviour?).

Reply:
@benfred We already have these kind of checks
For TF: https://github.com/NVIDIA-Merlin/Transformers4Rec/blob/main/tests/torch/conftest.py#L19
For PyT: https://github.com/NVIDIA-Merlin/Transformers4Rec/blob/main/tests/tf/conftest.py

However, running `pytest tests` and reaching those skipping imports makes the whole testing being skipped. I don't know why. May it be because of subfolders structure? I created an issue in the past: https://github.com/NVIDIA-Merlin/Transformers4Rec/issues/340

This is why I created the script specifying containers

Reply:
thanks for the clarification!